src/main/java/com/adyen/service/resource/binlookup/GetCostEstimate.java

```diff
@@ -0,0 +1,15 @@
+package com.adyen.service.resource.binlookup;
+
+import java.util.Arrays;
+import com.adyen.Client;
+import com.adyen.Service;
+import com.adyen.service.Resource;
+
+public class GetCostEstimate extends Resource {
+
+    public GetCostEstimate(Service service) {
+        super(service,
+                service.getClient().getConfig().getEndpoint() + "/pal/servlet/BinLookup/" + Client.BIN_LOOKUP_API_VERSION + "/getCostEstimate",
```

Comment:
same as above. constant for URL.

Reply:
Added constant in Client with the other urls